atomic_reactor/plugin.py

```diff
@@ -227,6 +221,26 @@ def run(self, keep_going=False, buildstep_phase=False):
                 plugin_is_allowed_to_fail = plugin_request['is_allowed_to_fail']
```

Comment:
can't this be simplified to:
```
plugin_is_allowed_to_fail = plugin_request.get('is_allowed_to_fail', True)
```

Reply:
Probably this, If I read correctly, it can be defined in two places
```python
plugin_is_allowed_to_fail = plugin_request.get('is_allowed_to_fail', getattr(plugin_class, "is_allowed_to_fail", True))
```